web/src/playground_links_popover.ts

```diff
@@ -128,6 +128,7 @@ function register_click_handlers(): void {
                     url_template.expand({code: extracted_code}),
                 );
             } else {
+                $view_in_playground_button.removeAttr("href");
```

Comment:
This is a good chance, but add a comment here explaining the bug it's preventing (probably a bit more concise than the commit message); that'll help keep the code readable for posterity.

Reply:
Added the following comment:
```js
// Remove the href attribute that was set when there was only
// one playground link, so that it doesn't interfere with the
// popover toggle when multiple playground links exist.
```